scripts/sync/support.rb

```diff
@@ -93,6 +93,33 @@ def absolutize_links(repository_slug, path_transformer, contents_source_path, so
     @lines = lines.collect { | line | UrlAbsolutizer.absolutize_links(line, repository_slug, path_transformer, contents_source_path, source_paths) }
   end
 
+  def escape_things_that_look_like_jsx_tags
+    in_code_block = false
+    @lines = lines.collect do | line |
+      in_code_block = !in_code_block if line.start_with?('```')
+
+      if !in_code_block
+        in_backticks = false
+        line.chars.collect do | char |
+          # Doesn't handle escaped backticks within backticks
+          in_backticks = !in_backticks if char == '`'
+
+          if !in_backticks
+            case char
```

Comment:
Could you just decode all the content?

Reply:
No - the < and > inside the backticks need to stay as literals, otherwise, they get rendered as &gt; and &lt;